components/bl-list-component/README.md

```diff
@@ -0,0 +1,55 @@
+# List
+
+List is the component that can be used in Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). It allows you to add a standard list to your application.
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="643"/>
+</p>
+
+Additional properties allow customizing the font size, background color, text color, padding and behavior of the component. Refer to the Properties section below.
+
+## Configuration
+
+The configuration can be done in the UI Builder or using the Codeless Logic. You need to select the type of list (ordered/unordered) and specify the data that will be displayed inside the list.
```

Comment:
Redundant description

You will add info about list types into at start of readme. Also helpful link to mdn will be good too.